### ISSUE TITLE
Cannot browse the list of shortcuts

### ISSUE THREAD

**Issue by myroslavarm** [guillep/pharo-core#6229]
**Description**
There are two parts of this bug but they are most likely related.
1. KeymapBrowser is broken.
2. When clicking on the triangle in top-right corner of the SystemBrowser, there is no option to look at shortcuts.

**To Reproduce**
For (1) go to System->click Keymap Browser->get a DNU error.
For (2) as already pointed out, you cannot find this option in the window menu, and another way is to get to the list of shortcuts programatically, which according to PharoEnvironmentHelp is listed to be
```{Smalltalk}
KMDescription new categories: #(NautilusGlobalShortcuts); openWithSpec
```
does not work either (the `KMDescription` class appears to be not implemented).

**Expected behavior**
One should be able to browse the list of shortcuts in Pharo. Or if this functionality is replaced by the menu which pops up after the right click in the SystemBrowser, then it has to be deleted completely.

**Version information:**
 - OS: Windows 10
 - Version: Pharo 9.0.0 (64 bit)

**Comment by myroslavarm** [comment]
After this is fixed, it would be nice to update https://github.com/pharo-project/pharo/issues/5952 as well, because the current instructions to get the list of shortcuts in Pharo env help point to invalid approaches.

**Comment by hernanmd** [comment]
Fixed in #15027
